converter/dicom-cast/samples/templates/deploy-with-healthcareapis.json

```diff
@@ -138,6 +138,8 @@
     "dicomServiceName": "[concat(parameters('workspaceName'), '/', parameters('dicomServiceName'))]",
     "fhirEndpoint": "[concat('https://', parameters('workspaceName'), '-', parameters('fhirServiceName'), '.fhir.azurehealthcareapis.com')]",
     "dicomWebEndpoint": "[concat('https://', parameters('workspaceName'), '-', parameters('dicomServiceName'), '.dicom.azurehealthcareapis.com')]",
+    "patientSystemId": "[parameters('patientSystemId')]",
```

Comment:
You are still missing parameters

Reply:
Removed the parameters from whole document